themes/bootstrap3/templates/record/cart-buttons.phtml

```diff
@@ -9,14 +9,15 @@
       <span class="cart-link-label btn-type-minus"><?=$this->transEsc('Remove from Book Bag') ?></span>
     </a>
     <noscript>
-      <form method="post" name="addForm" action="<?=$this->url('cart-processor')?>">
-        <input type="hidden" name="ids[]" value="<?=$this->escapeHtmlAttr($cartId)?>" />
-        <?php if ($cart->contains($cartId)): ?>
-          <input class="btn btn-default" type="submit" name="delete" value="<?=$this->transEscAttr('Remove from Book Bag')?>"/>
-        <?php else: ?>
-          <input class="btn btn-default" type="submit" name="add" value="<?=$this->transEscAttr('Add to Book Bag')?>"/>
-        <?php endif; ?>
-      </form>
+      <?php if (!$cart->contains($cartId)): ?>
```

Comment:
Exactly what problem is being solved here? I'm not too keen on GET links with side effects -- maybe it's better to just leave this alone for now and then remove the `<noscript>` entirely in VuFind 9.

Reply:
The problem is that you can't have a form inside noscript. Also, the buttons look seriously bad among the other links. I'll be happy to drop this one if you think it's better to keep the form thing.